cms/server/contest/handlers/contest.py

```diff
@@ -298,17 +310,11 @@ def _get_current_user_from_cookie(self):
         return participation
 
     def setup_locale(self):
-        self.all_translations = self.service.translations
-        lang_codes = list(iterkeys(self.all_translations))
-
-        if self.contest.allowed_localizations:
-            lang_codes = filter_language_codes(
-                lang_codes, self.contest.allowed_localizations)
+        lang_codes = list(iterkeys(self.available_translations))
 
         # Select the one the user likes most.
         basic_lang = 'en'
```

Comment:
DEFAULT_TRANSLATION.identifier?

Reply:
Will fix when I push since this is overwritten by a later commit.